packages/cli-ui/server/api/index.js

```diff
@@ -30,6 +30,22 @@ function api(message, client) {
         new ProjectApi(client).deleteProjectById(message.id)
     }
 
+    if(message.type === "GET_FAVORITE_PROJECTS") {
+        new ProjectApi(client).getFavoriteProjects()
```

Comment:
Maybe we need to create an instance of a class, instead of creating a collection of classes to access for each method?